package.json

```diff
@@ -111,19 +111,19 @@
     "cozy-authentication": "2.6.1",
     "cozy-bar": "7.16.0",
     "cozy-ci": "0.4.1",
-    "cozy-client": "23.0.0",
+    "cozy-client": "23.3.0",
     "cozy-client-js": "0.19.0",
     "cozy-device-helper": "1.12.0",
     "cozy-doctypes": "1.82.1",
     "cozy-flags": "2.6.0",
     "cozy-harvest-lib": "5.3.1",
     "cozy-logger": "1.6.0",
-    "cozy-pouch-link": "22.0.0",
+    "cozy-pouch-link": "23.3.0",
```

Comment:
don't upgrade the pouchlink yet. I'm working on using this upgrade to simplify the migration (https://github.com/cozy/cozy-drive/pull/2336)